main.cpp

```diff
@@ -1010,7 +901,375 @@ Part 1e - Step 19: Request a review
+    1) material (std::string)
+    2) number of breaks (int)
+    3) speed indicator in Km/h (int)
+    4) fuel indicator in percentage (int)
+    5) hour indicator (std::string)
+3 things it can do:
+    1) accelerate
+    2) brake
+    3) steer
+*/
+struct HandleBar
+{
+    //5 properties:
+    //    1) material (std::string)
+    std::string material = "plastic";
+    //    2) number of breaks (int)
+    int numberOfBreaks = 2;
+    //    3) speed indicator in Km/h (int)
+    int speedIndication = 0;
+    //    4) fuel indicator in percentage (int)
+    int fuelPercentage = 50;
+    //    5) hour indicator (std::string)
+    std::string hourIndication = "00:00:00";
+    //3 things it can do:
+    //    1) accelerate
+    int accelerate(int accelerationAmount); // return an updated speedIndication value given an acceleration amount as parameter
+    //    2) brake
+    int brakeOfAGivenAmount(int brakingAmount); // return an updated speedIndication value given braking amount as parameter
+    //    3) steer
+    void steer(std::string direction); // steer based on direction that could be for example right or left
+};
+
+/*
+Thing 6) Wheel
+5 properties:
+    1) material (std::string)
+    2) wheel rim type (std::string)
+    3) maximum pressure in bar (float)
+    4) colour (std::string)
+    5) percentage of wear (float)
+3 things it can do:
+    1) loose pressure
+    2) bounce
+    3) spin
+*/
+struct Wheel
+{
+    //5 properties:
+    //    1) material (std::string)
+    std::string material = "rubber";
+    //    2) wheel rim type (std::string)
+    std::string rimType = "aluminum";
+    //    3) maximum pressure in bar (float)
+    float maxBarPressure = 6.2f;
+    //    4) colour (std::string)
+    std::string colour = "black";
+    //    5) percentage of wear (float)
+    float wearPercentage = 0.0f;
+    //3 things it can do:
+    //    1) loose pressure
+    void loosePressure(float pressureAmount); // loose pressure of a given pressure amount
+    //    2) bounce
+    void bounce();
+    //    3) spin
+    void spin(int revolutionsPerMinute); // set the revolution per minute to make the wheel spinning
+};
+
+/*
+Thing 7) Seat
+5 properties:
+    1) material (std::string)
+    2) shape (std::string)
+    3) number (int)
+    4) colour (std::string)
+    5) percentage of cushioning (float)
+3 things it can do:
+    1) absorb vibrations
+    2) cover objects holder
+    3) comfort driver
+*/
+struct Seat
+{
+    //5 properties:
+    //    1) material (std::string)
+    std::string material = "leather";
+    //    2) shape (std::string)
+    std::string shape = "rounded";
+    //    3) number (int)
+    int numberOfSeats = 1;
+    //    4) colour (std::string)
+    std::string colour = "black";
+    //    5) percentage of cushioning (float)
+    float cushioningPercentage = 90.0f;
+    //3 things it can do:
+    //    1) absorb vibrations
+    void absorbVibrations();
+    //    2) cover objects holder
+    void coverObjectsHolder();
```

Comment:
How does a seat cover an object holder?
What is an object holder?
Pick better actions for a seat that make sense.  
- [ ] revise

Reply:
changed action, hoping the new one makes more sense than the last one.